crates/ide_completion/src/item.rs

```diff
@@ -143,24 +143,24 @@ pub struct CompletionRelevance {
     ///     f($0) // type of local matches the type of param
     /// }
     /// ```
-    pub exact_type_match: bool,
+    pub type_match: Option<CompletionRelevanceTypeMatch>,
     /// This is set in cases like these:
     ///
     /// ```
-    /// fn foo(bar: u32) {
-    ///     $0 // `bar` is local
-    /// }
-    /// ```
-    ///
-    /// ```
-    /// fn foo() {
-    ///     let bar = 0;
-    ///     $0 // `bar` is local
```

Comment:
This change is unrelated to this PR, but combining these two examples has been discussed as something we'd like to do.